README.md

```diff
@@ -147,3 +147,9 @@ spec:
 ### Important notes
 
 Crawler takes events/messages from intervals with startTimestamps >= "from" and < "to" of intervals.
+Crawler works in to modes: EVENTS and MESSAGES.
+
+####MESSAGES mode
+Crawler sends messages via gRPC request to Data-Processor(Processor). The Processor does with messages whatever it wants and sends response. </br>
+The response may contain mapping entries of stream to [MessageID](https://github.com/th2-net/th2-grpc-common/blob/4dd3aa2917fa1af683b6cd50ff6d250e652b6bb7/src/main/proto/th2_grpc_common/common.proto#L37).
```

Comment:
Please, add information about restrictions for those IDs: they should be from the current interval